snuba/clickhouse/columns.py

```diff
@@ -245,6 +245,20 @@ class DateTime(ColumnType):
     pass
 
 
+class Enum(ColumnType):
+    def __init__(self, values: Sequence[Tuple[str, int]]) -> None:
```

Comment:
Did you check and ensure enum values can only be ints ?

Reply:
Yes, I didn't see this in the docs specifically, but got this hint when I tried to create values with anything else.